scripts/release/status.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+const { GITHUB_REF, GITHUB_TOKEN } = process.env
+
+const TIMEOUT = 10 * 1000
+const MAX_ATTEMPTS = 30
+
+let attempts = 0
+
+async function checkStatuses (contexts) {
+  const url = `https://api.github.com/repos/DataDog/dd-trace-js/commits/${GITHUB_REF}/status`
+  const response = await fetch(url, {
+    headers: {
+      Accept: 'application/vnd.github+json',
+      Authorization: `Bearer ${GITHUB_TOKEN}`,
+      'X-GitHub-Api-Version': '2022-11-28'
+    }
+  })
+
+  if (response.status !== 200) {
+    console.log(response)
+    console.log(response.text())
+
+    throw new Error('Could not get status from GitHub.')
```

Comment:
I guess having it all in the error message might be nice?

```suggestion
    throw new Error(
      `Could not get status from GitHub.\n\n${util.inspect(response)}\n${util.inspect(response.text?.())}`
    )
```